src/apps/backend/modules/comment/internal/comment-util.ts

```diff
@@ -0,0 +1,15 @@
+import { Comment } from '../types';
+import { CommentDB } from './store/comment-db';
+
+export default class CommentUtil {
+  public static convertCommentDBToComment(commentDb: CommentDB): Comment {
+    const comment = new Comment();
+    comment.id = commentDb._id.toString();
+    comment.task = commentDb.task.toString();
+    comment.account = commentDb.account;
```

Comment:
`comment.account = commentDb.account` is not correct. We have to do it this way if the type of comment.account is an object ID then comment.account = comment.account.toString() else if it is an object then comment.account = AccountUtil.convertAccountDBToAccount(commentDb.account).